.circleci/config.yml

```diff
@@ -586,7 +586,7 @@ workflows:
           filters:
             branches:
               only:
-               - raft-tdp-main
+               - develop
```

Comment:
@abottoms-coder are we implementing the new strategy with the merge of this PR? if so, does `develop` already exist? will things break with this merge into `raft-tdp-main`?

Reply:
Yes, with the merge of this PR I will rename `raft-tdp-main` to `develop` so things should not break. I mentioned this in a comment on the main PR page.